infrastructure/modules/armonik/core-configmap.tf

```diff
@@ -25,6 +25,11 @@ resource "kubernetes_config_map" "core_config" {
     Redis__ClientName                          = "ArmoniK.Core"
     Redis__Ssl                                 = "true"
     Redis__SslHost                             = local.redis_ssl_host
+    S3__EndpointUrl                            = try(var.storage_endpoint_url.s3.url, "")
+    S3__Login                                  = try(var.storage_endpoint_url.s3.login, "")
+    S3__Password                               = try(var.storage_endpoint_url.s3.password, "")
+    S3__MustForcePathStyle                     = try(var.storage_endpoint_url.s3.must_force_path_style, false)
+    S3__BucketName                             = try(var.storage_endpoint_url.s3.bucket_name, "")
```

Comment:
In worker configmap, we use another naming for the S3 parameters (S3Storage__ServiceURL, S3Storage__AccessKeyId, S3Storage__SecretAccessKey, S3Storage__BucketName ). Can we standardize the naming for both core-configmap and worker-configmap?